admin-node-setup.sh

```diff
@@ -49,15 +49,61 @@ cp $tmp_dir/*.yaml $kube_dir/
 
 rm -rf $tmp_dir
 
-# First time setup of user-configuration for salt-master
+# Create CaaSP config dir
 if [ ! -d "/etc/caasp" ]; then
     mkdir /etc/caasp
 fi
```

Comment:
Shouldn't this be created by the package?

Reply:
Not on upgrade, tx update can't make any persisted changes to /etc

Reply:
I didn't realize about this until now. This will be tricky if we include a new package in the future that comes with its own `/etc` file and that file(s) are not managed by salt explicitly (because we don't want to tune them).

Reply:
Or if a system package updates its `/etc` config for let's say more secure defaults... guh.

Reply:
Okay, it's an overlay mount.